code/modules/religion/religion_sects.dm

```diff
@@ -445,25 +445,13 @@
 		return FALSE
 
 	var/mob/living/carbon/human/H = L
-	var/heal_amt = 40 //double healing, no chance to mess up, and shorter cooldown than default
-	var/heal_cost = 40
+	var/heal_amt = 20 //no chance to mess up
```

Comment:
Bible bashing isn't a competitive favor farm since no one lets you give them holy water first, no reason to make the bible literally worse than other sect options